spec/unit/util/execution_spec.rb

```diff
@@ -144,6 +155,24 @@ def stub_process_wait(exitstatus)
         call_exec_windows('test command', {}, @stdin, @stdout, @stderr)
       end
 
+      context 'cwd option' do
+        let(:cwd) { 'cwd' }
+        it "should execute the command in the specified working directory" do
+          Dir.expects(:chdir).with(cwd).yields
```

Comment:
Unit test updates for the block-style invocation. Note we need to put an expectation on Process.create now to ensure that we still call it with the right args since the method's logic has changed.